packages/wdio-allure-reporter/src/reporter.ts

```diff
@@ -503,7 +503,7 @@ export default class AllureReporter extends WDIOReporter {
 
         if (!disableWebdriverStepsReporting) {
             this.attachJSON('Response', commandResult)
-            this._endTest(AllureStatus.PASSED)
+            this.endStep(AllureStatus.PASSED)
```

Comment:
I don't see `this.endStep` being defined somewhere 🤔

Reply:
Here https://github.com/webdriverio/webdriverio/blob/3f678588dda3ade709f09bd0631f6b76d32ddaf0/packages/wdio-allure-reporter/src/reporter.ts#L795